osu.Game/Screens/Play/HUD/FailingLayer.cs

```diff
@@ -105,6 +107,28 @@ private void updateBindings()
                 enabled.Value = false;
         }
 
+        /// <summary>
+        /// Tries to fade based on "Fade playfield when health is low" setting
+        /// </summary>
+        /// <param name="fadeDuration">Duration of the fade</param>
+        /// <param name="easing">Type of easing</param>
+        /// <param name="fadeIn">True when you want to fade in, false when you want to fade out</param>
+        public void TryToFade(float fadeDuration, Easing easing, bool fadeIn)
+        {
+            if (HUDEnabled.Value)
+            {
+                if (fadeIn)
+                {
+                    if (enabled.Value)
```

Comment:
`enabled` should be renamed for legibility to be able to tell what setting toggles what.